test/config-next/ra.json

```diff
@@ -53,7 +53,11 @@
 				"pendingAuthzLifetime": "7h",
 				"validAuthzLifetime": "7h",
 				"orderLifetime": "7h",
-				"maxNames": 10
+				"maxNames": 10,
+				"identifierTypes": [
```

Comment:
I think the other profiles should get an explicit `identifierTypes` config key as well, since the "TODO(#8184): Remove this default." requires that we make it explicit for all profiles.